# Message for the sonar scans
Header header
# Scan information
float32 range # meters
float32 beam_angle # rads
float32 max_range # meters
float32 min_range # meters
